packages/grid/_modules_/grid/components/containers/GridRoot.tsx

```diff
@@ -12,23 +17,35 @@ export const GridRoot = React.forwardRef<HTMLDivElement, GridRootProps>(function
   props,
   ref,
 ) {
-  const { className, ...other } = props;
   const classes = useStyles();
-  const apiRef = React.useContext(GridApiContext);
+  const apiRef = React.useContext(GridApiContext)!;
+  const propsContext = React.useContext(GridPropsContext)!;
```

Comment:
The naming consistency seems questionable. Would it be clearer with?
```suggestion
  const apiRef = React.useContext(GridApiRefContext)!;
  const rootProps = React.useContext(GridRootPropsContext)!;
```